spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -133,7 +133,7 @@ module CCMS # rubocop:disable Metrics/ModuleLength
 
           it 'generates INJ_REASON_NO_WARNING_LETTER block with reason' do
```

Comment:
This could probably be tested in the test you added on line 824, then this one can be removed. Something like

```
        it 'should be hard coded with the correct notification' do
          attributes = [
            [:proceeding_merits, 'INJ_REASON_NO_WARNING_LETTER'],
            [:proceeding_merits, 'INJ_RECENT_INCIDENT_DETAIL'],
            [:global_merits, 'INJ_REASON_POLICE_NOT_NOTIFIED'],
            [:global_merits, 'INJ_REASON_NO_WARNING_LETTER']
          ]
          attributes.each do |entity_attribute_pair|
            entity, attribute = entity_attribute_pair
            block = XmlExtractor.call(xml, entity, attribute)
            expect(block).to have_text_response 'Apply Service application. See uploaded provider statement and report'
          end
        end
```